kernel/src/main/java/org/kframework/compile/SortCells.java

```diff
@@ -784,7 +784,7 @@ public void apply(KApply k) {
                 }
                 @Override
                 public void apply(KVariable var) {
-                    if (!inKCell && cellFragmentVars.containsKey(var)) {
+                    if (!inKCell) {
```

Comment:
What?! Why was this removed?

Reply:
From the [Java documentation](https://docs.oracle.com/javase/8/docs/api/java/util/HashMap.html#remove-java.lang.Object-), you don't need to check if a key is present before removing it from a `HashMap`.